tokio-postgres/src/connect_raw.rs

```diff
@@ -192,6 +194,16 @@ where
     }
 }
 
+fn no_channel_binding(config: &Config) -> Result<(), Error> {
+    match config.channel_binding {
+        config::ChannelBinding::Disable | config::ChannelBinding::Prefer => Ok(()),
+        config::ChannelBinding::Require => Err(Error::authentication(
+            "server did not use channel binding".into(),
+        )),
+        config::ChannelBinding::__NonExhaustive => unreachable!(),
```

Comment:
I don't understand the point of this. Is it a style thing? It seems like it circumvents the exhaustive-match check.

Reply:
That's the point - we want to be able to add new channel binding modes in the future (if necessary) without breaking backwards compatibility.